src/component/AddList.js

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import {useSelector} from "react-redux"
+
+const AddList=()=>{
+    const lists= useSelector((state)=>{ 
```

Comment:
since your state/ store is only an array you can just return the state

`const lists = useSelector(state =>state)`